src/NHibernate/Engine/RowSelection.cs

```diff
@@ -62,5 +63,11 @@ public bool DefinesLimits
 		{
 			get { return maxRows != NoValue || firstRow > 0; }
 		}
+
+		public string Hint
```

Comment:
This is a strange place to add query hint...

Reply:
What would be a better place?